src/storage/mvcc/txn.rs

```diff
@@ -58,9 +60,15 @@ impl<S: Snapshot> MvccTxn<S> {
             start_ts,
             writes: vec![],
             write_size: 0,
+            collapse_rollback: true,
         }
     }
 
+    pub fn collapse_rollback(mut self, collapse: bool) -> Self {
+        self.collapse_rollback = collapse;
```

Comment:
When would `collapse_rollback` be false? seems only in tests? So could we adjust the tests since `collapse_rollback` would never be set to false? Or add it into config to make it optional?

Reply:
Some tests need to test the case with many rollbacks.